src/components/Lognin/out/log.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { destroySession, TOKENKEY } from 'util/auth';
+
+const Log = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+    destroySession();
+    window.location.reload();
+  };
+
+  useEffect(() => {
+    if (localStorage.getItem(TOKENKEY)) {
+      setIsLoggedIn(true);
+    }
+  }, [isLoggedIn, handleLogout]);
+
+  if (isLoggedIn) {
+    return (
+      <div className=" mr-4 md:mr-12">
+        <button className="relative inline-flex items-center justify-center p-0.5 mb-2 md:mr-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-cyan-500 to-blue-500 group-hover:from-cyan-500 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-cyan-200 dark:focus:ring-cyan-800" type="submit" onClick={handleLogout}>
+          <span className="relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0">
+            Log Out
+          </span>
+        </button>
+      </div>
+    );
+  }
+  return (
+    <div className="mr-4 md:mr-12">
+      <button className="relative inline-flex items-center justify-center p-0.5 mb-2 md:mr-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-cyan-500 to-blue-500 group-hover:from-cyan-500 group-hover:to-blue-500 hover:text-white dark:text-white focus:ring-4 focus:outline-none focus:ring-cyan-200 dark:focus:ring-cyan-800" type="submit">
+        <a href="http:/login" className="relative px-5 py-2.5 transition-all ease-in duration-75 bg-white dark:bg-gray-900 rounded-md group-hover:bg-opacity-0">
+          Login
+        </a>
+      </button>
+    </div>
```

Comment:
[OPTIONAL]
Maybe you can consider abstracting this and making a button component